### PR TITLE
Pc 342

### PR DESCRIPTION
**Pull request by matrix-drorgo**
image

image

### CODE REVIEW

File: src/Components/Forms/DiagnosisAndRecommendations.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  formatDate,
+  languageDirection,
+  history,
+  verticalName,
+}) => {
+  return <React.Fragment>DiagnosisAndRecommendations</React.Fragment>;
+};
+
+const mapStateToProps = (state) => {
+  return {
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations.js:18]: missing in the props the permission mode

File: src/Components/Forms/DiagnosisAndRecommendations.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    patient: state.active.activePatient,
+    encounter: state.active.activeEncounter,
+    languageDirection: state.settings.lang_dir,
+    formatDate: state.settings.format_date,
+    verticalName: state.settings.clinikal_vertical,
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations.js:23]: why do we need the vertical name?

File: src/Components/Forms/MedicalAdmission.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  languageDirection,
+  history,
+  verticalName,
+}) => {
+  return <React.Fragment>MedicalAdmission</React.Fragment>;
+};
+
+const mapStateToProps = (state) => {
+  return {
+    patient: state.active.activePatient,
```
Comment (matrix-amiel) [inline src/Components/Forms/MedicalAdmission.js:19]: as the comment in the DiagnosisAndRecommendations component

File: src/Components/Forms/TestsAndTreatments.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  languageDirection,
+  history,
+  verticalName,
+}) => {
+  return <React.Fragment>TestsAndTretments</React.Fragment>;
+};
+
+const mapStateToProps = (state) => {
+  return {
+    patient: state.active.activePatient,
```
Comment (matrix-amiel) [inline src/Components/Forms/TestsAndTreatments.js:19]: as the comment in the DiagnosisAndRecommendations component

File: src/Components/Generic/EncounterSheet/EncounterForms/FormsContainer.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+function a11yProps(index) {
```
Comment (matrix-amiel) [inline src/Components/Generic/EncounterSheet/EncounterForms/FormsContainer.js:39]: what is the mean of the function name- a11yProps?

File: src/Components/Generic/EncounterSheet/EncounterForms/index.js [hunk 6]
```diff
@@ -1,14 +1,35 @@
 import React, { useEffect, useState } from 'react';
 import StyledPatientFiles from './Style';
-import { Button } from '@material-ui/core';
+import { getForms } from '../../../../Utils/Services/API';
```
Comment (matrix-amiel) [inline src/Components/Generic/EncounterSheet/EncounterForms/index.js:3]: change to absolute path

File: src/Components/Forms/DiagnosisAndRecommendations.js (hunk 1 above)
Comment (matrix-drorgo) [reply src/Components/Forms/DiagnosisAndRecommendations.js:18]: not relevent

File: src/Components/Forms/DiagnosisAndRecommendations.js (hunk 2 above)
Comment (matrix-drorgo) [reply src/Components/Forms/DiagnosisAndRecommendations.js:23]: not relevent

File: src/Components/Forms/MedicalAdmission.js (hunk 3 above)
Comment (matrix-drorgo) [reply src/Components/Forms/MedicalAdmission.js:19]: not relevent

File: src/Components/Forms/TestsAndTreatments.js (hunk 4 above)
Comment (matrix-drorgo) [reply src/Components/Forms/TestsAndTreatments.js:19]: not relevant

File: src/Components/Generic/EncounterSheet/EncounterForms/FormsContainer.js (hunk 5 above)
Comment (matrix-drorgo) [reply src/Components/Generic/EncounterSheet/EncounterForms/FormsContainer.js:39]: not relevant

File: src/Components/Generic/EncounterSheet/EncounterForms/index.js (hunk 6 above)
Comment (matrix-drorgo) [reply src/Components/Generic/EncounterSheet/EncounterForms/index.js]: resolved

**Pull request merged**